WordPress/src/main/java/org/wordpress/android/datasets/SiteSettingsTable.java

```diff
@@ -29,6 +30,7 @@
     public static void createTable(SQLiteDatabase db) {
         if (db != null) {
             db.execSQL(SiteSettingsModel.CREATE_SETTINGS_TABLE_SQL);
+            db.execSQL(JetpackSettingsModel.CREATE_JP_SETTINGS_TABLE_SQL);
```

Comment:
Any specific reason we are handling 2 tables in a single file? This might throw off some people. Also, if we are making the separation, wouldn't it be better to do it in FluxC? I haven't thought about this too much, so I am actually asking and not suggesting :)